etc/docker/helm/rucio-server/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+name: rucio-server
+version: 0.1.0
```

Comment:
should this reflect the rucio version? or is it separate?

Reply:
There is a set of metadata one can use in the Chart.yaml. Some are required and some are optional. `version` is mandatory and it should reflect the version of the chart and not of the application it installs. For that there is another field `appVersion` which is necessary. But I deliberately omitted that one, as the Rucio version can be changed by setting the `image.tag` value without the need to update the chart itself.